site/docs/operations/cluster-sizing.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Cluster Sizing
+id: cluster-sizing
+---
+
+Jet cluster performance depends on multiple factors including the pipeline design and user defined functions. Therefore, planning the Jet cluster remains a complex task that requires a knowledge of the Jet architecture and concepts. We will introduce a basic guideline that will help you size your cluster. 
+
+We recommend to always benchmark your setup before deploying it to production. See a sample cluster sizing with benchmark that can be used as a reference.
+
+Please read the Hazelcast IMDG Deployment and Operations Guide when using the [local IMDG](concepts/in-memory-storage.md#relationship-with-hazelcast-imdg) setup. Your Jet cluster will run both workloads (local IMDG and Jet jobs) so you should plan for it.
```

Comment:
I'm not sure of this "local IMDG" term. It sounds like IMDG is something else running side by side, when it's one and the same.